### PR TITLE
feat: Force Multiplier Program page

### LINKED ISSUE #20: Build the Force Multiplier Program page
Page URL: `/training-and-education/force-multiplier-program`

Includes content:
- page header
- static content - intro, course cards, sidebar

Mirrors https://github.com/USSF-ORBIT/ussf-portal-mvp/blob/main/pages/training__force-multiplier-program.md

### PR DESCRIPTION
**Pull request by abbyoung**
## Description 

* Adds rebuild of MVP Force Multiplier Program page.
* Content and styling should match the current deployed FMP page
* Adds unit and integration tests

**Note**: This PR is branched from #58, so I'll merge that PR first and clean up the diff.

Fixes #20 

## Review Notes

Font Awesome Pro strikes again! Two of the shape icons on the page, the triangle and hexagon, are both inexplicably Pro level icons.

Because these feature prominently, I tried to find a workaround. While we don't have an auth token/registry setup for Pro, the MVP repo does have all the required files to use the Pro icons.

To try and reduce the bulk, I copied over only the essential files and was able to use the icons as `<i>` elements.

Lmk what you think + if there's another solution.

### CODE REVIEW

File: src/pages/training-and-education/force-multiplier-program.tsx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            Training and Education
+          </span>
+          <h1 className="page-header">Become Digitally Fluent</h1>
+          <p className="usa-intro text-light font-heading-lg">
+            The Digital University Force Multiplier Program
+          </p>
+          <LinkTo
+            className="usa-button usa-button--big external-link--alt text-white text-no-underline"
+            href="http://digitalu.udemy.com">
+            Register
```
Comment (suzubara) [inline src/pages/training-and-education/force-multiplier-program.tsx:19]: I noticed this button doesn't have the same hover state as on the MVP site, not a huge deal but maybe worth just looking into?

File: src/pages/training-and-education/force-multiplier-program.tsx [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                <p className="usa-intro">
+                  To be successful in space operations, we must all acquire and
+                  use cutting-edge digital skills. This service-wide program
+                  introduces and builds digital fluency in four key areas.
+                </p>
+                <Grid row className="margin-top-8 training">
+                  <Grid
+                    tablet={{ col: 8 }}
+                    className="usa-prose bg-gray-cool-3 padding-3">
+                    {/* <div className="tablet:grid-col-8 usa-prose bg-gray-cool-3 padding-3"> */}
```
Comment (suzubara) [inline src/pages/training-and-education/force-multiplier-program.tsx:38]: can this line be removed?

**Review by suzubara — COMMENTED** [review]
Just a minor thing, I think the button hover state CSS needs to be ported over too!

File: src/pages/training-and-education/force-multiplier-program.tsx (hunk 1 above)
Comment (abbyoung) [reply src/pages/training-and-education/force-multiplier-program.tsx:19]: great catch! thank you!

**Review by suzubara — APPROVED** [review]

**Pull request merged**